packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -147,6 +147,12 @@ class BottomNavigationBar extends StatefulWidget {
   /// The [iconSize], [selectedFontSize], [unselectedFontSize], and [elevation]
   /// arguments must be non-null and non-negative.
   ///
+  /// The [selectedLabelStyle.color] and [unselectedLabelStyle.color] values
```

Comment:
Is it possible to have the values like `selectedItemColor` get preference but if null, use the ones from the style? That feels more like a pattern we've seen before.

Reply:
Yes, done!